lib/zlib.js

```diff
@@ -54,8 +53,8 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
-  var ckey = ckeys[ck];
+
+for (const ckey of ckeys) {
```

Comment:
While `for-of` is the more modern approach, traditional `for-loop` is still by far the fastest which is why we have not migrated these. In order to go this direction, we would need to show the benchmark impact.